cmake/treedata/freebsd/subdirs.txt

```diff
@@ -3,8 +3,9 @@ xbmc/cores/VideoPlayer/Process/rbpi cores/VideoPlayer/Process/rbpi
 xbmc/input/touch                    input/touch
 xbmc/input/touch/generic            input/touch/generic
 xbmc/platform/freebsd               platform/freebsd
+xbmc/platform/freebsd/network       platform/freebsd/network
 xbmc/platform/linux/input           platform/linux/input
-xbmc/platform/linux/network         platform/linux/network
+xbmc/platform/linux/network/zeroconf platform/linux/network/zeroconf
```

Comment:
Move this line to `cmake/treedata/optional/common/avahi.txt` and add ` # AVAHI` at the end of the line.

Reply:
good idea :+1: